docs/kb/object-field-references.md

```diff
@@ -1946,7 +1946,7 @@ Visitor Activities may have the following values for
 <li>2 - View</li>
 <li>3 - Error</li>
 <li>4 - Success</li>
-<li>5 - Session</li>
+<li>5 - Visitor</li>
```

Comment:
I'm not comfortable changing the existing values because I'm not sure how or why it changed. Have you done any research on why the value in the docs is different?

Reply:
As far as I can tell this should have always been Visitor (committed in 2013) while this was added to api-docs as session in 2014

Reply:
My worry is that it was a product reason for changing it to "Session" in the docs or a technical reason that we couldn't change the existing value to "Session". By changing it now, we may undo that decision.

Reply:
We occasionally refer to a 'Visitor Session' in other documents and articles:
https://help.salesforce.com/articleView?id=pardot_activities_that-dont_trigger_a_sync_from_pardot_to_salesforce.htm&type=5

Our glossary defines 'Visitor' as the visitor session however:
https://help.salesforce.com/articleView?id=pardot_basics_glossary.htm&type=5

Reply:
@jpearson-sfdc agree with product possibility. I'd recommend removing it as well and we can follow up separately

Reply:
Sounds good! I will change it back to Session.